### ISSUE TITLE
[BUG] JS Snippets not working anymore.

### ISSUE THREAD

**Issue by MathieuTricoire** [cloudflare/workers-rs#408]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What version of `workers-rs` are you using?

0.0.18

### What version of `wrangler` are you using?

3.15.0

### Describe the bug

Hi everyone,

I've recently started working with Cloudflare and Cloudflare Workers, and I've encountered what appears to be a regression with "JS Snippets". Upon reviewing the repository's history, it seems that this feature was previously functioning correctly, but I suspect changes, potentially around the "glue" code, might have affected this functionality.

Error when running `npm run dev`
<img width="1029" alt="Screenshot 2023-11-07 at 4 03 14 PM" src="https://github.com/cloudflare/workers-rs/assets/255766/a3272c44-a531-4ab3-8f63-af5946dd120d">

WebAssembly code:
<img width="1019" alt="Screenshot 2023-11-07 at 4 03 42 PM" src="https://github.com/cloudflare/workers-rs/assets/255766/228ee45e-8a90-4153-bd37-5ced8104a13a">

Shim.mjs code (glue code):
<img width="869" alt="Screenshot 2023-11-07 at 4 04 03 PM" src="https://github.com/cloudflare/workers-rs/assets/255766/7730f6ef-23b8-4f5a-8b61-d2daeeaf9d5d">

Let me know if I miss something or if this is a real issue.

Thanks,
Mat

### Steps To Reproduce

Use my example: https://github.com/MathieuTricoire/workers-rs/tree/fix-js-snippets/examples/js_snippets

If you clone my repo and want to be sure to reproduce the error do not forget to replace the `wrangler.toml` build command with something like this, since my repo contains a fix:

```
[build]
command = "worker-build --release"
```

**Comment by MathieuTricoire** [comment]
FYI: I have a potential fix on my fork, let me know if you want me to create a PR.

**Comment by kflansburg** [comment]
I'm not familiar with this mechanism, is this part of wasm-bindgen that we need to support in our custom glue code? I know that we treat WebAssembly imports different than other runtimes which may be the issue here. Are you using Miniflare 2 or 3?

**Comment by MathieuTricoire** [comment]
Yes it's in wasm-bindgen: https://rustwasm.github.io/wasm-bindgen/reference/js-snippets.html and it was previously supported see this PR #142.

I'm using Miniflare 3.
